.github/workflows/ci.yml

```diff
@@ -80,7 +80,7 @@ jobs:
         run: wget -q -c ${{ inputs.flink_url }} -O - | tar -xz
 
       - name: Compile and test flink-connector-kafka
-        timeout-minutes: 30
+        timeout-minutes: 45
```

Comment:
e2e tests were timing out

Reply:
You should rebase, so this file becomes obsolete (we're re-using the same CI setup across the different externalized connectors). Timeouts are there set to 50 mins for this step and 60 mins for the entire workflow.

Reply:
Nice, removed!